src/ado2gh/GithubApiFactory.cs

```diff
@@ -15,15 +17,11 @@ public GithubApiFactory(OctoLogger octoLogger, HttpClient client, EnvironmentVar
             _environmentVariableProvider = environmentVariableProvider;
         }
 
-        public virtual GithubApi Create()
-        {
-            return Create("https://api.github.com");
-        }
-
-        public virtual GithubApi Create(string apiUrl)
+        public virtual GithubApi Create(string apiUrl = null, string personalAccessToken = null)
         {
-            var githubPat = _environmentVariableProvider.GithubPersonalAccessToken();
-            var githubClient = new GithubClient(_octoLogger, _client, githubPat);
+            apiUrl ??= DEFAULT_API_URL;
```

Comment:
Cool, so this should allow us to eventually do GHES migrations?

Reply:
I guess it should be supported right now even 😃